src/entities/vaultStrategy.ts

```diff
@@ -0,0 +1,28 @@
+import { Address } from "@graphprotocol/graph-ts";
+import { VaultStrategy } from "../../generated/schema";
+import { VaultStrategy as VaultStrategyABI } from "../../generated/VaultFactory/VaultStrategy";
+import { VAULT_FACTORY_ADDRESS } from "../constants";
+
+export function createVaultStrategy(
+  vaultStrategyAddress: Address
+): VaultStrategy {
+  const vaultStrategy = new VaultStrategy(vaultStrategyAddress.toHexString());
+  const vaultContract = VaultStrategyABI.bind(vaultStrategyAddress);
+
+  vaultStrategy.factory = VAULT_FACTORY_ADDRESS.toHexString();
+  vaultStrategy.vault = vaultContract.try_getVault().value.toHexString();
+  vaultStrategy.lbPair = vaultContract.try_getPair().value.toHexString();
+  vaultStrategy.operator = vaultContract.try_getOperator().value.toHexString();
+  vaultStrategy.strategistFee = vaultContract.try_getStrategistFee().value;
```

Comment:
can we add these fields to the `Vault` entity and load them from there instead of making contract calls?

Reply:
this function is called in `handleStrategyCreated` (when the factory creates a new strategy) so we don't know anything about the vault yet
it's like what we have in `createLBPair`, let me know if I'm missing something

Reply:
Ah got it I thought `vaultContract` was the `Vault` contract but it looks like the calls are being made from `VaultStrategy`. A nit but how about we rename to `vaultStrategyContract`?

Reply:
Yeah it was called vaultContract instead of vaultStrategyContract, I changed it